app/grandchallenge/evaluation/models.py

```diff
@@ -1164,13 +1185,32 @@ def save(self, *args, **kwargs):
                     actor_only=False,
                     send_action=False,
                 )
-            e = create_evaluation.signature(
-                kwargs={"submission_pk": self.pk}, immutable=True
-            )
-            on_commit(e.apply_async)
+            if self.phase.external_evaluation:
+                Evaluation.objects.create(submission=self)
```

Comment:
Even though it might not be necessary speed-wise it might be better to keep it all in the `create_evaluation` task so that it is logically handled in one place, that way future developers will only need to look in one place to find the business logic.